about.html

```diff
@@ -10,59 +10,96 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<div class="container"> <img src="img/lambda-black.png" alt="toplogo">
+
+<header>
+<nav>
+  <a href="index.html">Home</a>
+  <a href="#">About</a>
+  <a href="#">Products</a>
+  <a href="#">Blog</a>
+  <a href="#">Contact</a>
+</nav>
+</header>
+</div>
+
+<section class="top-content">
+<div class="png">
+<img src="img/jumbo-about.png"/>
+</div>
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+    <div class="container">
+
+      <section class="top-content2">
+          <h2>About</h2>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to <br> 
+             corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+           </div>
+          </section>  
     
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
+             
+<div id="section">
+<div class="strategy">
+  <img src="img/about-plan.png" alt="strategy">
```

Comment:
You have the right idea but maybe forgot about it, the alt attribute in each img tag is set to "strategy". For assessability purposes, it's recommended that the attribute reflect the sections more accurately.